src/google/protobuf/any.pb.cc

```diff
@@ -99,7 +99,7 @@ class Any::_Internal {
 Any::Any(::PROTOBUF_NAMESPACE_ID::Arena* arena,
                          bool is_message_owned)
   : ::PROTOBUF_NAMESPACE_ID::Message(arena, is_message_owned),
-  _any_metadata_(&type_url_, &value_) {
+  _any_metadata_(arena, &type_url_, &value_) {
```

Comment:
NB:

Local regeneration of any.pb.cc from any.proto removes some `PROTOBUF_EXPORT` statements from any.pb.h.
I did not included these changes into PR.